package.json

```diff
@@ -8,7 +8,7 @@
   },
   "dependencies": {
     "del": "^5.1.0",
-    "digitalmarketplace-frameworks": "github:alphagov/digitalmarketplace-frameworks#v17.8.2",
+    "digitalmarketplace-frameworks": "github:alphagov/digitalmarketplace-frameworks#remove-ons-link",
```

Comment:
```suggestion
    "digitalmarketplace-frameworks": "github:alphagov/digitalmarketplace-frameworks#17.8.2",
```

Reply:
good spot! Looks like some local stuff got erroneously committed. I will tidy it up.